test/tools/ossfuzz/protoToYul.h

```diff
@@ -40,14 +40,16 @@ class ProtoConverter
 	{
 		m_funcVars = std::vector<std::vector<std::vector<std::string>>>{};
 		m_globalVars = std::vector<std::vector<std::string>>{};
-		m_inForBodyScope = false;
+		m_inGenericForBodyScope = false;
+		m_inBoundedForBodyScope = false;
 		m_inForInitScope = false;
 		m_numNestedForLoops = 0;
 		m_counter = 0;
 		m_inputSize = 0;
 		m_inFunctionDef = false;
 		m_objectId = 0;
 		m_isObject = false;
+		m_forInitScopeExtEnabled = true;
```

Comment:
This is needed because scope extension is limited to variables declared directly inside for-init block, not nested blocks (see https://github.com/ethereum/solidity/pull/7747)